### ISSUE TITLE
NTR: Anti-Sm-autoantibody positivity

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#5798]
**Preferred term label:**
Anti-Sm-autoantibody positivity

**Synonyms**
Anti-Smith-autoantibody

**Definition (free text, please give PubMed ID)**
The presence of autoantibodies in the serum that react to seven proteins that consist of a core of small nuclear ribonucleoprotein (snRNP) particles.

Comment: Anti-Sm-autoantibodies are highly specific for systemic lupus erythematosus (SLE).

PMID: 31552434
DOI: 10.1007/s00296-019-04445-y

**Parent term (use hpo.jax.org/app)**
Autoimmune antibody positivity HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:536   (Disorder)   Systemic lupus erythematosus

**Your nano-attribution (ORCID)**
lmu: chrapp, @mgriese, @katrinknoflach

**Comment by pnrobinson** [comment]
added as HP:0033040
